src/main/java/com/google/sps/model/Event.java

```diff
@@ -3,13 +3,14 @@
 import java.util.Date;
 import java.util.List;
 import java.lang.Long;
+import java.util.UUID;
 
 public class Event implements Comparable<Event>{
-    private long eventId;
+    private UUID eventId;
```

Comment:
Can you do a toString from the UUID and use that as the key? Otherwise
you can get a hash from the UUID and use that as id. That way it will be a long everywhere.